app/views/bright_events/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="form-wrap">
+  <%= link_to bright_events_path, class: 'btn btn-return' do %>
+    <i class="uil uil-arrow-left"></i>
+  <% end %>  
+  <div class="form-container">
+    <h1>Create Event</h1>
+    <%= form_for (@event) do |f| %>
+      <%= f.text_field :title, placeholder: 'Título', value: @event.title, required: true %>
+      <%= f.text_field :description, placeholder: 'Descripción', value: @event.description, required: true %>
+      <%= f.date_field :date, placeholder: 'Fecha', value: @event.date, min: 0.days.ago, required: true %>
+      <%= f.text_field :location, placeholder: 'Ubicación', value: @event.location, required: true %>
+      <%= f.number_field :cost, in: 0.00..1000000.00, step: 0.01, placeholder: 'Precio', value: @event.cost, required: true %>
+      <%= f.submit 'Guardar cambios', class: 'btn btn-submit btn-warning', required: true %>
+    <% end %>
+  </div>
+</div>
```

Comment:
Have you considered doing a partial for the form since is the same code here and in the new ???
Maybe do it in another pr

Reply:
little tricky, but already implemented :+1: